qa/suites/deepsea/tier4/upgrade/3-test-phase.yaml

```diff
@@ -19,7 +19,8 @@ tasks:
         client.salt_master:
                 - 'deepsea --version || true'
                 - 'salt-run deepsea.version 2>/dev/null || true'
-                - 'salt-run status.report 2>/dev/null'
+                - 'salt-run upgrade.status 2>/dev/null'
+                - 'sudo zypper -n --no-gpg-checks install --force --no-recommends deepsea-cli'
```

Comment:
`exec` already runs as root, I believe, so you can drop the `sudo` here?

Reply:
see https://github.com/ceph/teuthology/blob/master/teuthology/task/exec.py#L48-L55

The `sudo` is redundant.

Reply:
also, you might as well write out `--non-interactive` (instead of `-n`)

IMO it's good practice, in code, to use long option names whenever possible

Reply:
Done :)